### ISSUE TITLE
Set the next map, Did not press the next round button, Why switch maps directly?

### ISSUE THREAD

**Issue by gpxjordan** [Myrcon/Procon-1#193]
My PRoCon version is 1.5.2.0:
https://github.com/Myrcon/Procon-1/pull/192

Sometimes the round is still in progress, When I just want to set, I want to play the next round of the map, Tickets are not zero, I don't know why PRoCon will immediately switch to the next round, Can someone tell me why this is?

**Comment by ColColonCleaner** [comment]
If you remove the current map from the maplist it will cancel the current round. This is not something procon specific, it's how the game operates.

**Comment by gpxjordan** [comment]
I did not delete the map in the "maplist" list, I just want to use "Set as next map" during the game, When the ticket does not become 0, the game immediately becomes the next round.

This happened to me, which shocked me, But can't this problem be fixed?

Because there is a job that needs to be busy recently, Too late to get back to you, please forgive me.

**Comment by ColColonCleaner** [comment]
That doesn't happen when I do that on the servers I have access to. I would suggest trying the 'ultimate map manager' plugin to see if changing the next map also stops the current round there.

**Comment by gpxjordan** [comment]
Please let me confirm again, Do you mean that you do the same thing with me and will not happen the same?

Please forgive me because Google is difficult to translate what you mean, So I need to confirm again.

**Comment by ColColonCleaner** [comment]
Correct. When I use the 'set as next map' button, it does not cause the round to end.

**Comment by gpxjordan** [comment]
I wanted to change my way of expression, When you do the same action with me, Your server will not immediately change a map? (Not the end round)

**Comment by ColColonCleaner** [comment]
Correct.

**Comment by gpxjordan** [comment]
I think I need a little time, then confirm to see if the same problem will occur.

Thank you for your reply!
